sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -1,10 +1,16 @@
 # Changelog
 
-### Coming Soon... 12.0.0-preview.2
+### 2019.08 Version 12.0.0-preview.2
 
 - [Breaking] Aborter class is no longer exposed from the package. You are expected to use
   the package [@azure/abort-controller](https://www.npmjs.com/package/@azure/abort-controller) to pass an abort signal to any of the async operations.
   `AbortController.timeout(<milliseconds>)` can be utitlized as an abort signal.
+- Union type of credential is updated to `{SharedKeyCredential | AnonymousCredential | TokenCredential}` in all the client constructors.
```

Comment:
Changelogs should focus on user impact rather than implementation details. Consider something like "generalized the credential parameter type in client constructors to support more credential types."